app/RefBank.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class RefBank extends Model
+{
+    protected $fillable = [
+        'account_vendor', 'account_number', 'id',
```

Comment:
please remove id form fillable "because id is auto increments so id is auto fillable"

Reply:
done..